src/components/index.njk

```diff
@@ -3,6 +3,15 @@ title: Components
 sortOrder: 4
 ---
 
-Components are reusable parts of the user interface that have been made to support a variety of applications.
+Components are the building blocks for any user interface.
 
-Individual components can be used in multiple different [patterns](/patterns) and contexts. For example, the [text input](/components/input) component can be used to ask for an [email](/components/input#email) address, annual salary, or someone’s name.
+Individual components can be used for various patterns and contexts. For example, the input component can be used to ask for an email address, a date or a name.
+
+Each component has guidance on:
+- when to use it
+- how to use it
+- variations you can use
+- how to check it for errors
+- user research done on it
+
```

Comment:
Probably doesn't make much difference so I wont block the PR on this but just for consistency we should probably use the list component for this and the list in the pattern file